src/component/Restaurants.js

```diff
@@ -63,9 +93,45 @@ function Restaurants (props) {
                     </select>
                 </div>
             </form>
-
+            {restaurantList}
         </div>
     );
 }
 
+Restaurants.propTypes = {
+    selected: PropTypes.func.isRequired
+};
+
 export { Restaurants };
+
+const DEFAULT_SELECT_VALUE = "";
+const IMAGE_SRC_MAP = Object.freeze({
+    "node_modules/place-my-order-assets/images/1-thumbnail.jpg": thumbnail1Src,
+    "node_modules/place-my-order-assets/images/2-thumbnail.jpg": thumbnail2Src,
+    "node_modules/place-my-order-assets/images/3-thumbnail.jpg": thumbnail3Src,
+    "node_modules/place-my-order-assets/images/4-thumbnail.jpg": thumbnail4Src
+});
+
+async function apiFetch(url) {
```

Comment:
I'm not sure the component should be making this request anyway.  I feel like this is maybe overly coupling the UI to the data it's fetching

Reply:
This has been moved to a separate module.